test/fixtures/test/forms.json

```diff
@@ -9,5 +9,26 @@
     "options": {
       "read_only": true
     }
+  },
+  {
+    "id": "33333",
+    "name": "Script and Reducers",
+    "options": {
+      "context": [
+        "return { foo() { return 'foo'; } }"
+      ],
+      "reducers": [
+        "formSubmission.storyTime = foo()\nreturn formSubmission"
+      ]
+    }
+  },
+  {
+    "id": "44444",
+    "name": "Reducer Error",
+    "options": {
+      "reducers": [
+        "syntaxError('foo;"
```

Comment:
@zfixler @nmajor25 regarding our recent conversation.  The point of this PR is that the code in this reducer currently would just throw a console warning instead of an error.  Not a terrible problem for very simple logic as we'll catch stuff, but for anything mildly complicated we could be getting errors in production that we neglected to test but no one currently is aware.